apps/src/music/views/MusicLabView.tsx

```diff
@@ -412,6 +422,14 @@ const MusicLabView: React.FunctionComponent<MusicLabViewProps> = ({
               }
               headerClassName={moduleStyles.headerWithBorder}
             >
+              {viewingOldVersion && (
+                <div
+                  id="viewingOldVersionBanner"
+                  className={moduleStyles.warningBanner}
+                >
+                  {musicI18n.viewingOldVersion()}
+                </div>
```

Comment:
Is it possible to use the [DSCO Alert](https://code-dot-org.github.io/code-dot-org/component-library-storybook/?path=/docs/designsystem-alert--docs) component here to be consistent with the other labs? Here's an example: https://github.com/code-dot-org/code-dot-org/blob/86d844eb70bcb7cb4eac73db476002770b8d39b5/apps/src/codebridge/Workspace/Workspace.tsx#L99-L106